components/prize_pool/wikis/pubg/prize_pool_award_custom.lua

```diff
@@ -0,0 +1,30 @@
+---
+-- @Liquipedia
+-- wiki=pubg
+-- page=Module:PrizePool/Award/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Lua = require('Module:Lua')
+local Variables = require('Module:Variables')
+
+local AwardPrizePool = Lua.import('Module:PrizePool/Award', {requireDevIfEnabled = true})
+
+local CustomAwardPrizePool = {}
+
+-- Template entry point
+function CustomAwardPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	args.localcurrency = args.localcurrency or Variables.varDefault('tournament_currency')
```

Comment:
Do these wikis acutally make use of this functionality todaty?

Reply:
they also have it in their ppt custom so i think yes